### ISSUE TITLE
Memories not work

### ISSUE THREAD

**Issue by ychy00001** [Josh-XT/AGiXT#467]
### Description

Use Learning function update file with two line txt msg, memories save success. Then use normal chat self.agent.memories.context_agent not found anything.

### Steps to Reproduce the Bug

1. go to Learning page
2. upload txt file  like: "Piter is a engineer"
3. go to chat page, not select smart chat
4. with message "What is Piter's occupation"
5. click send message

### Expected Behavior

when process chat task , format_prompt with learning info.

### Actual Behavior

I print some log， like this， show Memory collection count : 0, collection_list []
```
 def context_agent(self, query: str, top_results_num: int) -> List[str]:
        print(f"Memory query: {query}, limit: {top_results_num}")
        if not self.chroma_client:
            self.chroma_client = self.initialize_chroma_client()
            self.collection = self.get_or_create_collection()
        count = self.collection.count()
        collection_list = self.collection.peek()
        print(f"Memory collection count: {count}")
        print(f"Memory collection list: {collection_list}")
```

### Operating System

- [ ] Microsoft Windows
- [X] Apple MacOS
- [ ] Linux
- [ ] Android
- [ ] iOS
- [ ] Other

### Python Version

- [ ] Python <= 3.9
- [X] Python 3.10
- [ ] Python 3.11

### Environment Type - Connection

- [X] Local
- [ ] Remote

### Environment Type - Container

- [X] Using Docker
- [ ] Not Using Docker

### Acknowledgements

- [X] My issue title is concise, descriptive, and in title casing.
- [X] I have searched the existing issues to make sure this bug has not been reported yet.
- [X] I am using the latest version of AGiXT.
- [X] I have provided enough information for the maintainers to reproduce and diagnose the issue.

**Comment by godofdream** [comment]
```
agixt-agixt-1  | /agixt/agixt/AGiXT.py:209: RuntimeWarning: coroutine 'Memories.store_result' was never awaited
agixt-agixt-1  |   pass
```
seems the code doesn't wait for the async

**Comment by Josh-XT** [comment]
Resolved in #552
